### PR TITLE
Incorporating changes as per FFF suggestions (1,3,4)

### PR DESCRIPTION
**Pull request by surajsingla333**

### CODE REVIEW

**Comment by bhaskarSingh** [comment]
are we going to remove the pairs chapter as per the feedback by FFF?

**Comment by surajsingla333** [comment]
> 
> 
> are we going to remove the pairs chapter as per the feedback by FFF?

No, previously we defined pair as `power_move = sp.pair("Bullet_seed", 95)`. This declaration is also correct but we can declare it like a simple tuple of 2 elements, like `power_move = ("Bullet_seed", 95)` and that's what we will be doing now.

File: lessons/introduction-to-smartpy/04/04.mdx [hunk 1]
```diff
@@ -40,4 +40,4 @@ This contract calls an initialization method `self.init`, contained in `sp.Contr
 
 2) Pass a parameter `name` in the initialization function,
 
-3) Declare a state variable `name` of type `sp.TString` for your class and initialize it will the class parameter `name`
+3) Declare a state variable `name` and initialize it will the class parameter `name`. In this case, it's of type `sp.TString`
```
Comment (bhaskarSingh) [inline lessons/introduction-to-smartpy/04/04.mdx]: It's ambiguous, simplify the 3rd point

Comment (surajsingla333) [reply lessons/introduction-to-smartpy/04/04.mdx]: Okay.

**Pull request closed without merging**